org.sonarlint.eclipse.core.tests/src/test/java/org/sonarlint/eclipse/core/internal/engine/connected/ConnectedEngineFacadeManagerTest.java

```diff
@@ -31,21 +31,21 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.fail;
-import static org.sonarlint.eclipse.core.internal.server.ServersManager.AUTH_ATTRIBUTE;
-import static org.sonarlint.eclipse.core.internal.server.ServersManager.ORG_ATTRIBUTE;
-import static org.sonarlint.eclipse.core.internal.server.ServersManager.PASSWORD_ATTRIBUTE;
-import static org.sonarlint.eclipse.core.internal.server.ServersManager.PREF_SERVERS;
-import static org.sonarlint.eclipse.core.internal.server.ServersManager.URL_ATTRIBUTE;
-import static org.sonarlint.eclipse.core.internal.server.ServersManager.USERNAME_ATTRIBUTE;
+import static org.sonarlint.eclipse.core.internal.engine.connected.ConnectedEngineFacadeManager.AUTH_ATTRIBUTE;
+import static org.sonarlint.eclipse.core.internal.engine.connected.ConnectedEngineFacadeManager.ORG_ATTRIBUTE;
+import static org.sonarlint.eclipse.core.internal.engine.connected.ConnectedEngineFacadeManager.PASSWORD_ATTRIBUTE;
+import static org.sonarlint.eclipse.core.internal.engine.connected.ConnectedEngineFacadeManager.PREF_SERVERS;
+import static org.sonarlint.eclipse.core.internal.engine.connected.ConnectedEngineFacadeManager.URL_ATTRIBUTE;
+import static org.sonarlint.eclipse.core.internal.engine.connected.ConnectedEngineFacadeManager.USERNAME_ATTRIBUTE;
 
 /**
  * To have this test pass when launched from the IDE, you have to set -pluginCustomization argument (see pom.xml).
  */
-public class ServersManagerTest {
+public class ConnectedEngineFacadeManagerTest {
 
   private static final ISecurePreferences ROOT_SECURE = SecurePreferencesFactory.getDefault().node(SonarLintCorePlugin.PLUGIN_ID);
   private static final IEclipsePreferences ROOT = InstanceScope.INSTANCE.getNode(SonarLintCorePlugin.PLUGIN_ID);
-  private ServersManager manager;
+  private ConnectedEngineFacadeManager manager;
```

Comment:
What about `ConnectionsManager` ? I usually don't like the term manager on a class because it is vague, in this case it might be ok